telegram/src/main/kotlin/com/github/kotlintelegrambot/Bot.kt

```diff
@@ -203,20 +203,8 @@ class Bot private constructor(
 
     fun getMe() = apiClient.getMe().call()
 
-    /**
-     * Use this method to send text messages
-     * @param chatId unique identifier for the target chat
-     * @param text text of the message to be sent, 1-4096 characters after entities parsing
-     * @param parseMode mode for parsing entities in the message text
-     * @param disableWebPagePreview disables link previews for links in this message
-     * @param disableNotification sends the message silently - users will receive a notification with no sound
-     * @param replyToMessageId if the message is a reply, ID of the original message
-     * @param replyMarkup additional options - inline keyboard, custom reply keyboard, instructions to remove reply
-     * keyboard or to force a reply from the user
-     * @return the sent [Message] on success
-     */
```

Comment:
I think we should keep the documentation in this method, and just update the description of the `chatId` parameter to reflect that now it can be either a chat id or a channel username:

```kotlin
* @param chatId Unique identifier for the target chat or username of the target channel (in
* the format @channelusername)
```

Reply:
👍  updated param description in other methods also